javascript/classes.json

```diff
@@ -160,6 +160,62 @@
           "deprecated": false
         }
       },
+      "class_static_initialization_blocks": {
```

Comment:
Note the inventive name :-)

Reply:
Let's omit "class" since this feature is nested under classes anyway. And then it'll sort nicely with the other `static` stuff. (This suggestion will fail the tests, unfortunately—you'll have to `npm run fix` I think)

```suggestion
      "static_initialization_blocks": {
```